backend/apps/owasp/api/search/chapter.py

```diff
@@ -39,11 +40,11 @@ def chapters(request):
     """Search chapters API endpoint."""
     page = int(request.GET.get("page", 1))
     query = request.GET.get("q", "")
-
+    active_chapters_count = IndexSynonymsMixin.get_algolia_index_count("local_chapters")
     chapters = get_chapters(query=query, page=page, meta=request.META)
     return JsonResponse(
         {
-            "active_chapters_count": Chapter.active_chapters_count(),
+            "active_chapters_count": active_chapters_count,
```

Comment:
You can inline the value, like:
```suggestion
            "active_chapters_count": IndexBase.get_total_count("chapters")
```